app/src/main/res/layout/fragment_library.xml

```diff
@@ -29,6 +29,7 @@
         layout="@layout/item_playlist"
         android:layout_width="0dp"
         android:layout_height="wrap_content"
+        android:visibility="gone"
```

Comment:
1.Hide Liked and Local albums
2. I think in such case it is better to leave Deprecation message empty